index.js

```diff
@@ -0,0 +1,80 @@
+const apiUrl = 'https://ghibliapi.herokuapp.com';
+let localMovieReviewStorage = {};
+
+document.addEventListener('DOMContentLoaded', () => {
+	seedSelectBox();
+	let selectBox = document.querySelector('.movie-titles');
+	selectBox.addEventListener('change', getMovieInfo);
+	let form = document.querySelector('.user-review');
+	form.addEventListener('submit', makeNewReview);
+});
+
+async function seedSelectBox(){
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
+	let defaultOption = document.createElement('option');
+	selectBox.appendChild(defaultOption);
+	for(let i = 0; i < movies.data.length; i++){
+		let option = document.createElement('option');
+		option.text = movies.data[i].title;
+		localMovieReviewStorage[movies.data[i].title] = [];
+		selectBox.appendChild(option);
+	}
+}
+
+async function getMovieInfo(){
+	let descriptionBoard = document.querySelector('.movie-description');
+	let child = descriptionBoard.firstElementChild;
+	//removes all child elements from the board
+	while(child){
+		descriptionBoard.removeChild(child);
+		child = descriptionBoard.firstElementChild;
+	}
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
+	let selectedMovie = movies.data[selectBox.selectedIndex - 1];
+	let title = document.createElement('h3');
+	title.classList.add('title')
+	title.innerText = selectedMovie.title;
+	let releaseYear = document.createElement('strong');
+	releaseYear.innerText = selectedMovie.release_date;
+	let description = document.createElement('p');
+	description.innerText = selectedMovie.description;
+	descriptionBoard.appendChild(title);
+	descriptionBoard.appendChild(releaseYear);
+	descriptionBoard.appendChild(description);
+	displayMovieReviews(selectedMovie.title);
+}
```

Comment:
Your function ⬆️  is doing 3 things. 
    - Clearing the previous info if any
    - Fetching the new info
    - Displaying/appending to the DOM the new info
This is fine, but a good practice would have been to do the above in separate functions and just call them here. I would really help whomever is reading your code.